definitions/webbrowser.yaml

```diff
@@ -273,6 +273,7 @@ sources:
     paths:
       - '%%users.localappdata%%\Mozilla\Firefox\Profiles\*\places.sqlite'
       - '%%users.appdata%%\Mozilla\Firefox\Profiles\*\places.sqlite'
+      - '%%users.appdata%%\Roaming\Mozilla\Firefox\Profiles\*\places.sqlite'
```

Comment:
GRR documentation agrees, the roaming doesn't need to be there:
https://github.com/google/grr/blob/master/grr/proto/knowledge_base.proto#L136